packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -403,6 +543,38 @@ See [ID Token Claims](/docs/reference/api/oidc/#id-token-claims) for a list of I
 }
 ```
 
+You can also use the `remove` operation to remove members from existing JSON objects and elements from existing arrays. For example, you have an array that contains the user's preferred airports, and you want to remove an airport from the array.
+
+Existing target JSON object:
+```json
```

Comment:
what about example with removing JSON property?

Reply:
Yes we can add one.